runtime/services/lightd/flora.js

```diff
@@ -30,7 +30,7 @@ Flora.prototype.handlers = {
   },
   'rokid.turen.end_voice': function (msg) {
     logger.log('rokid.turen.end_voice')
-    this.light.stopFile('@yoda', SETPICKUPURI)
+    this.light.stopFile('lothal-launcher', SETPICKUPURI)
```

Comment:
`lothal-launcher` seems not a base application :(

Reply:
Do I need to monitor the event "rokid.turen.end_voice" in the app who's appid is "lothal-launche"?